boards/arduino-nano-33-iot/Makefile.features

```diff
@@ -14,6 +14,7 @@ FEATURES_PROVIDED += periph_usbdev
 
 # Put other features for this board (in alphabetical order)
 FEATURES_PROVIDED += highlevel_stdio
+FEATURES_PROVIDED += tinyusb_device
```

Comment:
Defining the feature `tinyusb_device` in addition to the `highlevel_stdio` feature produces a feature conflict. Boards that don't enable any other `stdio_*` module, enable the `stdio_cdc_acm` module by default which in turn requires the features `highlevel_stdio` and `usbus_cdc_acm`. `usbus_cdc_acm` requires feature `periph_usbdev` so that it is not possible to use feature `tinyusb_device` and `highlevel_stdio` at the same time.

Reply:
My plan was to add an additional `stdio_tinyusb` module to be able to use tinyUSB and the USB interface for the stdio.

Reply:
Due to the same problem for a couple of nRF52 boards, I enabled feature `tinyusb_device` only for boards that don't use the USB port for the stdio.

Reply:
Yup, you're right ! I'll removed then for now.

Reply:
It's the same for all the other boards that provide the `highlevel_stdio` feature.

Reply:
Removed.